### DISCUSSION TITLE
Translations via Crowdin

### DISCUSSION THREAD

**Discussion by andrew-codechimp** [andrew-codechimp/HA-Battery-Notes#940] [Announcements]
Battery Notes is now integrated with Crowdin to make translations far easier 🎉 
If you want to get Battery Notes in your own language here's [everything you need to know](https://andrew-codechimp.github.io/HA-Battery-Notes/translations/).

**Comment by comradekingu** [comment]
Happy to help if you use Weblate. Crowdin messes up the locales for my language, and it is the wrong tool for the job.

**Reply by andrew-codechimp** [reply]
I don't have the time to setup/host/maintain my own Webplate, Crowdin does a pretty good job for a small project.

You can add translations by adding a new PR with the file here
https://github.com/andrew-codechimp/HA-Battery-Notes/tree/main/custom_components/battery_notes/translations

If you want me to create a starter file please let me know the language/locale you want.
